js/ui/scheduler/header/calendar.js

```diff
@@ -81,9 +81,15 @@ export default class SchedulerCalendar extends Widget {
         return result;
     }
 
+    _optionChanged(args) {
+        if(args.name === 'value') {
+            this._calendar?.option('value', args.value);
+        }
+    }
```

Comment:
Minor comment.

Maybe will it be better to use a switch here?
With the switch add new options to this method will be simpler:
```suggestion
    _optionChanged({ name, value }) {
        switch(name) {
          case 'value':
            this._calendar?.option('value', value);
            break;
          default:
            break;
        }
    }
```